github-page/_docs/types/returnModelType.md

```diff
@@ -13,7 +13,7 @@ class Kitten {
   @prop()
   public name?: string;
 
-  public static findByName(this: ReturnModelType<Kitten>, name: string) { // this is an Instance Method
+  public static findByName(this: ReturnModelType<typeof Kitten>, name: string):  Promise<Kitten> { // this is an Instance Method
```

Comment:
```suggestion
  public static findByName(this: ReturnModelType<typeof Kitten>, name: string):  Promise<DocumentType<Kitten>[] | undefined> { // this is an Instance Method
```

sorry, but it does **not** return  an instance of the class, it returns the found documents (and returns them as an array), that is why i didnt type-def the returns value, because the "infered one is better"

Reply:
Will adjust it then. The missing return type causes TS2742 though.

Reply:
> Will adjust it then. The missing return type causes TS2742 though.

could you give me how you use it to give this? i wouldnt know a case where the infered one would not work... (best would be discord to post this)

Reply:
Will try to. Have to extract the code - will take a bit.

Reply:
sry, was a pebcak - didn't migrate completely and still had the ModelType<T> & T thingy in there. Removed it and I'm fine. 
Will update the PR to remove this and sign ig.